### DISCUSSION TITLE
Help me decide on which icon to use

### DISCUSSION THREAD

**Discussion by slipx06** [slipx06/sunsynk-power-flow-card#181] [Polls]
![image](https://github.com/slipx06/sunsynk-power-flow-card/assets/7227275/71b65557-bbef-4d6e-a53b-73fef979b316)

**Comment by slipx06** [comment]
Thanks. Leaning towards this
![image](https://github.com/slipx06/sunsynk-power-flow-card/assets/7227275/3f8c6623-5467-4992-ab28-65792433a102) 
![image](https://github.com/slipx06/sunsynk-power-flow-card/assets/7227275/22803994-1e34-4b63-ae7d-635328c2519e)

**Reply by Huntedza** [reply]
Prefer the left since it matches the Icon of the PV inputs
![image](https://github.com/slipx06/sunsynk-power-flow-card/assets/21158862/b980e35e-ed9d-466d-ad60-65199db5bed7)

**Comment by slipx06** [comment]
![image](https://github.com/slipx06/sunsynk-power-flow-card/assets/7227275/9a86fe03-4236-4064-9e59-8630b3f7d146)

**Comment by slipx06** [comment]
The updated image in included in release 4.6.1. Thanks for the input
